Strategy11/formidable-forms
# Enhancing Field Settings Accessibility

This PR addresses various accessibility issues in field settings to enhance usability with screen readers and keyboard navigation.

## Related Issue:
[Improve field settings accessibility](https://github.com/Strategy11/formidable-pro/issues/4233)

## Testing Instructions:
1. Make sure you have the Formidable PRO and Formidable Surveys plugins:
    - **Formidable PRO PR**: https://github.com/Strategy11/formidable-pro/pull/4383
    - **Formidable Surveys PR**: https://github.com/Strategy11/formidable-surveys/pull/221
2.  Open an existing form or create a new one and add, for example, a Text Field.
3. Check the accessibility of settings and make sure the settings work well with a screen reader.

## Note:
I made a video showing how I use the settings with the VoiceOver on my Mac. Let me know if you see ways to make it better.

https://www.loom.com/share/2fbb1c4fb8804109abf49f6fee8f4bed?sid=3edfb1d2-847c-4e9d-84e9-ca617607708b
